rust/connlib/clients/shared/src/lib.rs

```diff
@@ -221,7 +226,7 @@ where
                         }
                     }
                     if let Some(t) = exponential_backoff.next_backoff() {
-                        tracing::error!("Connection to portal failed. Retrying connection to portal in {} seconds", t.as_secs());
+                        tracing::error!("Connection to portal failed. Retrying connection to portal in {} milliseconds", t.as_millis());
```

Comment:
This was printing `0` to the logs in many cases

Reply:
Nit: If it's not machine-readable either way, just doing the debug print `{:?}` will automatically put a reasonable unit like `s`, `ms`, or `us` (with the mu Unicode character)

Reply:
Since we are already changing this line, I propose we change it to a `debug` level.

Reply:
👍